README.md

```diff
@@ -61,6 +61,40 @@ Here's how to use it:
     ```js
     import Snackbar from 'react-native-snackbar';
     ```
+## Manual installation
+
+In case step 2 above throws errors:
+
+2. insert the following lines of code:
+
+In `android/settings.gradle`:
+```
```

Comment:
So we get syntax highlighting, add java after ```.

E.g. \```java

You only need to add it to the opening of the code blocks. So do it for three code blocks